docs/sql/statements/create-repository.rst

```diff
@@ -584,6 +584,105 @@ Parameters
   The port number of the proxy.
 
 
+.. _sql-create-repo-gcs:
+
+``gcs``
+-------
+
+A ``gcs`` repository stores snapshots on the `Google Cloud Storage`_ service.
+
+Parameters
+''''''''''
+
+.. _sql-create-repo-gcs-account:
+
+**bucket**
+  | *Type:* ``text``
+  | *Required*
+
+  Name of the `Google Cloud Storage`_ bucket used for storing snapshots.
+  The bucket must already exist before the repository is created.
+
+  .. NOTE::
+
+    CrateDB masks this parameter. You cannot query the parameter value from
+    the :ref:`sys.repositories <sys-repositories>` table.
```

Comment:
```suggestion
    This parameter will be masked (shown as ``[xxxxx]``) when querying
    :ref:`sys.repositories <sys-repositories>`.
```

Would be shorter and at the same time explain more concretely what is happening.
Same for the notes below